packages/components/src/styles/footnotes/footnotes.stories.ts

```diff
@@ -55,39 +52,42 @@ export const Default = {
 };
 
 /**
- * Use the class with a `ul` element to display the footnotes as an unordered list.
+ * Use the alternative html elements for alternative appearances:
+ *
+ * - unnumbered: use a `ul` element to create an unnumbered list.
+ * - numbered: use an `ol` element to create a numbered list.
+ * Usually start with "1", use the html `start` attribute to set the starting number of the list.
```

Comment:
This phrase feel off. I feel we can remove the initial part `Usually start with "1"`

Reply:
Maybe here we can have: "numbered: is the default variant"

Reply:
What about: The html `start` attribute to set the starting number of the list, the default value is '1'.